contrib/nydusify/pkg/viewer/viewer.go

```diff
@@ -61,7 +61,7 @@ func New(opt Opt) (*FsViewer, error) {
 	}
 
 	mode := "cached"
-	digestValidate := true
+	digestValidate := false
```

Comment:
Why change here?

Reply:
In this place, my thoughts--the popular version is rafs v6, which not support Digest validate. I change it to false, which can reduce the times of detection.
I have observe the logic of this code, there is something conflicted. And i want to change it to as the followings ,but i don't know is this a good idea  in terms of the evaluation of rafs's version (in the last pr, i just keep it `digestValidate := true` )
```
	mode := "cached"
	digestValidate := false
	if opt.FsVersion != "6" {
		mode = "direct"
		digestValidate = true
	}
```

Reply:
Thanks, we can decide the `DigestValidate` value later, so let's remove the `digestValidate := false` here?